### ISSUE TITLE
Some question

### ISSUE THREAD

**Issue by ThomasXDA** [el0xren/Samsung_Additional_Features#9]
It was great.
 Is there a tweak for (not cutting off the internet when making a call)?
CSC
Thank u.

**Comment by oldman20** [comment]
i dont think so, but in setting maybe has option switch network from sim 1 (during calling) to sim 2. I remember that
